### PR TITLE
graph-toolkit-one-productivity-hub

### PR DESCRIPTION
**Pull request by aycabas**
Added One Productivity Hub sample under the following folder: graph-toolkit-one-productivity-hub.

### CODE REVIEW

File: graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "appName": "OneProductivityHub",
+    "projectId": "392be947-8ed3-4f14-8dd6-b1e4878ac4bc",
+    "version": "2.1.0",
+    "isFromSample": false,
```
Comment (SLdragon) [inline graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json:5]: Remove `projectId`, and change  `isFromSample` to `true`

File: graph-toolkit-one-productivity-hub/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    "devDependencies": {
+        "@microsoft/teamsfx-cli": "1.*"
+    },
+    "license": "MIT",
+    "dependencies": {
+        "@microsoft/mgt": "^2.5.2",
+        "@microsoft/mgt-element": "^2.5.2",
+        "@microsoft/mgt-react": "^2.5.2",
+        "@microsoft/mgt-teamsfx-provider": "^2.5.2"
```
Comment (SLdragon) [inline graph-toolkit-one-productivity-hub/package.json:19]: Move these dependencies to tabs/package.json file

File: graph-toolkit-one-productivity-hub/tabs/src/components/Tab.jsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    this.setState({
+      showLoginPage: consentNeeded
+    });
+    Providers.globalProvider.setState(consentNeeded ? ProviderState.SignedOut : ProviderState.SignedIn);
+    return consentNeeded;
+
+  }
+
+ 
+
```
Comment (SLdragon) [inline graph-toolkit-one-productivity-hub/tabs/src/components/Tab.jsx:59]: Format the code and remove unnecessary line break

File: graph-toolkit-one-productivity-hub/README.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+## Try the Sample with Visual Studio Code Extension
+
+### Local Debug the Sample
+
+1. Clone the repo to your local workspace or directly download the source code.
+1. Download [Visual Studio Code](https://code.visualstudio.com) and install [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit).
+1. Open the project in Visual Studio Code.
+1. Open Debug View (`Ctrl+Shift+D`) and select "Debug (Edge)" or "Debug (Chrome)" in dropdown list.
+1. Press "F5" to open a browser window and then select your package to view contact exporter sample app.
```
Comment (SLdragon) [inline graph-toolkit-one-productivity-hub/README.md:30]: typo,  -> to view one productivity hub app

**Comment by SLdragon** [comment]
@aycabas ,  thank you very much for your contribution to our sample gallery, I leave some comments for your PR~

File: graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json (hunk 1 above)
Comment (HuihuiWu-Microsoft) [inline graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json:5]: "isFromSample" should be set to true so telemetry event can be correctly queried.

File: graph-toolkit-one-productivity-hub/README.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+> If you don't have an Azure subscription, create [a free account](https://azure.microsoft.com/en-us/free/) before you begin
+
+1. Install [Node.js](https://nodejs.org/en/download/) (use the latest v14 LTS release)
+1. To install the TeamsFx CLI, use the npm package manager:
+
+    ```
+    npm install -g @microsoft/teamsfx-cli
+    ```
+
+1. Clone this sample project.
```
Comment (HuihuiWu-Microsoft) [inline graph-toolkit-one-productivity-hub/README.md:51]: I think better tell users how to clone this sample with CLI as other samples do:
teamsfx new template graph-toolkit-one-productivity-hub
[Note] The CLI command is teamsfx new template <sample_id>,<sample_id> here is the same with your sample folder name.

File: graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json (hunk 1 above)
Comment (aycabas) [reply graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json:5]: done.

File: graph-toolkit-one-productivity-hub/package.json (hunk 2 above)
Comment (aycabas) [reply graph-toolkit-one-productivity-hub/package.json:19]: done.

File: graph-toolkit-one-productivity-hub/tabs/src/components/Tab.jsx (hunk 3 above)
Comment (aycabas) [reply graph-toolkit-one-productivity-hub/tabs/src/components/Tab.jsx:59]: done.

File: graph-toolkit-one-productivity-hub/README.md (hunk 4 above)
Comment (aycabas) [reply graph-toolkit-one-productivity-hub/README.md:30]: done.

File: graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json (hunk 1 above)
Comment (aycabas) [reply graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json:5]: done.

File: graph-toolkit-one-productivity-hub/README.md (hunk 5 above)
Comment (aycabas) [reply graph-toolkit-one-productivity-hub/README.md:51]: Done.

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

File: graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json [hunk 6]
```diff
@@ -1,8 +1,8 @@
 {
     "appName": "OneProductivityHub",
-    "projectId": "392be947-8ed3-4f14-8dd6-b1e4878ac4bc",
+    "projectId": "",
```
Comment (SLdragon) [inline graph-toolkit-one-productivity-hub/.fx/configs/projectSettings.json:3]: remove projectid

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by zyxiaoyuer — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Pull request merged**